Makefile

```diff
@@ -48,21 +46,27 @@ clone-tuned:
 	  cd $(TUNED_DIR) && git checkout $(TUNED_COMMIT) && cd .. && \
 	  rm -rf $(TUNED_DIR)/.git)
 
-build: $(BINDATA) pkg/generated
+build: $(BINDATA) generate-deepcopy pkg/generated
```

Comment:
Perhaps at some point add `performance-addon-operator-bin` target?

Reply:
For sure, I will do it once will add PAO dockerfiles.